docs/recipes/Internationalizing-Your-Module.md

```diff
@@ -2,6 +2,250 @@
 
 # Internationalizing Your Module
 
-Recipe is forthcoming.
+Internationalization with [react-intl](https://github.com/formatjs/react-intl) is quick and simple in One App.
+
+*If you used the [One App Module Generator](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module) to create the module some of these steps may already be completed.*
+
+## Component Creation
+Begin with a simple parent and child component.
+
+`Parent.jsx`
+```es6
+import React from 'react';
+import Child from './Child';
+
+const Parent = () => (
+  <Child />
+);
+
+export default Parent;
+```
+
+`Child.jsx`
+```es6
+import React from 'React';
+
+const Child = () => <div>greeting</div>;
+
+export default Child;
+```
+
+## Module Locale Files
+
+Next create the locale files. [one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) will bundle the locale files as a part of [one-app-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler). The README provides more information on the structure of locale files.
+
+Create a folder structure and content as follows:
+```
+- locale
+  - en-US.json
+  - es-MX.json
+```
+
+`en-US.json`
+```json
+{
+  "greeting": "Welcome!"
+}
+```
+
+`es-MX.json`
+```json
+{
+  "greeting": "Bienvenidos!"
+}
+```
```

Comment:
NOTE: section maybe needed. 

@10xLaCroixDrinker is this still applicable?

The labels `isLoading`, `lastError`, `error`, `locale`, and `links` are reserved and may not be used for application data.

I thought we had a fix for that ...

Reply:
@jsmartfo I'm confused. The only value for the locale files is the key `greeting`.

Reply:
I believe that is fixed in v4.0.0 of one-app-ducks. the shape of state was changed so that locale data does not mix with the loading data/

Reply:
Yes that was fixed

Reply:
Do we want @brantmullinix to mention that in this documentation?

Reply:
If it is fixed, not sure we need to mention it. Might just confuse people.